internal/pkg/beskar/tls.go

```diff
@@ -5,15 +5,14 @@
 package beskar
 
 import (
-	"context"
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
 	"os"
 	"strings"
 
 	"github.com/distribution/distribution/v3/configuration"
-	dcontext "github.com/distribution/distribution/v3/context"
+	"github.com/sirupsen/logrus"
```

Comment:
All reference to `dcontext` logger have been updated accordingly, this is due to a changes in github.com/distribution/distribution/v3, the `dcontext` package has been moved as an internal package now unreachable from this code, and `dcontext` logger is using `logrus`, so that should be fine